### ISSUE TITLE
YouTube Preview Image & Title before downloading

### ISSUE THREAD

**Issue by studiofruits** [imputnet/cobalt#1004]
### describe the feature you'd like to see

It would be great if a youtube video thubnail and title previewed upon entering a youtube url in the input. This can be easily done using oEmbed e.g https://www.youtube.com/oembed?url=https://www.youtube.com/watch?v=jYeEBAWXELk&format=json OR if thats too much hassel you can just preview the video thumbnail using the video id https://i.ytimg.com/vi/jYeEBAWXELk/hqdefault.jpg

**Comment by elouangrimm** [comment]
Yeah, like it could pop up below the text box and show the title, author, and the thumbnail.
